tests/test-tag-and-attribute-sanitizer.php

```diff
@@ -680,6 +680,11 @@ public function get_body_data() {
 				'<a class="foo" href="">value</a>',
 			),
 
+			'a_with_wrong_host'                                         => array(
+				'<a class="foo" href="http://foo bar">value</a>',
+				'<a class="foo" href="">value</a>',
+			),
+
```

Comment:
Please add test cases for examples found in https://github.com/Automattic/amp-wp/issues/977#issuecomment-369230272